scm/driver/github/testdata/pr.json

```diff
@@ -371,7 +371,7 @@
         "site_admin": false
     },
     "merge_commit_sha": "e5bd3914e2e596debea16f433f57875b5b90bcd6",
-    "merged": false,
+    "merged": true,
```

Comment:
the json payload in `pr.json` is for an open pull request;  therefore it should not be set to `merged: true`.  I would instead suggest adding a new json, real world json file and creating a new test case.

Reply:
I tested on my own repo, the `merged` and `merge_at` are always marked or filled in the case of merged-pr. And both not marked or filled in the case of closed-pr and opening-pr.

Reply:
I think there may be a misunderstanding.  The `pr.json` file is for an open pull request (the `state` field is set to `open`) which means it cannot have merged set to true.  The `pr.json` file is a valid, real world json and this pull request makes the file invalid.

Reply:
Hmmm, it the `pr.json` is for a opening-pr, then its `merge_at` must be `null`. If it is not merged (opening), how can it have a merge time... So I think something in the `pr.json` must be wrong.

But having another test json (maybe `pr_merged.json`) make sense.